ui/pages/bridge/hooks/useSubmitBridgeTransaction.ts

```diff
@@ -20,6 +20,7 @@ export default function useSubmitBridgeTransaction() {
   const submitBridgeTransaction = async (
     quoteResponse: QuoteResponse & QuoteMetadata,
   ) => {
+    // TODO catch errors and emit ActionFailed here
```

Comment:
is this still relevant?

Reply:
It is! The event needs to be added later